src/python/py/models/README.md

```diff
@@ -172,6 +172,18 @@ python3 -m onnxruntime_genai.models.builder -i path_to_local_folder_on_disk -o p
 python3 builder.py -i path_to_local_folder_on_disk -o path_to_output_folder -p precision -e execution_provider -c cache_dir_to_store_temp_files --extra_options enable_cuda_graph=1
 ```
 
+#### Use 8 Bits Quantization in QMoE
```

Comment:
Can you add a link to this section after `Enable Cuda Graph`?

https://github.com/microsoft/onnxruntime-genai/blob/a477802a411cc1ab77f5c23834ca807081e20ed9/src/python/py/models/README.md?plain=1#L15-L19